R/character.R

```diff
@@ -32,33 +63,32 @@ lime.character <- function(x, model, preprocess, tokenization = default_tokenize
     stop(m_type, ' models are not supported yet', call. = FALSE)
   )
 
-  function(cases, labels, n_labels = NULL, n_features, n_permutations = 5000, dist_fun = 'euclidean', feature_select = 'auto') {
+  function(cases = x, labels = NULL, n_labels = NULL, n_features, n_permutations = 5000, feature_select = 'auto') {
```

Comment:
cases shouldn't be set to the training data? The idea is that you create an explainer once for a model and then call it whenever you want to explain the output of the model...

Reply:
In text, there is no need to prepare data so there is no notion of training data. 
The options are:
- introducing a fake notion of training data, the purpose would be to copy the behavior of DF ;
- keep things the way they are.

Let me know what you prefer, but we would keep in mind whatever we choose it will change with the change of API to come.

Reply:
I know - I prefer 1. still :-)

Reply:
I removed the default parameter as requested.
FYI in the current API of lime.character, the `x` parameter is a fake parameter which is never evaluated but both DF and character API are now similar.